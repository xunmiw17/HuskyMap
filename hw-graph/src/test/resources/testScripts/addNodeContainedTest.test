# Create a graph
CreateGraph graph1

# Add three nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add an already existing node
AddNode graph1 n1

# Print the nodes in the graph
ListNodes graph1

# Add two already existing nodes, and a new node
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

# Print the nodes in the graph
ListNodes graph1