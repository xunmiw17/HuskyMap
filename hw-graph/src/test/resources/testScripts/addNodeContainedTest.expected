# Create a graph
created graph graph1

# Add three nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add an already existing node
added node n1 to graph1

# Print the nodes in the graph
graph1 contains: n1 n2 n3

# Add two already existing nodes, and a new node
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

# Print the nodes in the graph
graph1 contains: n1 n2 n3 n4